From e17278e1da0d2481ff5a4bc764af9cd88e12ebd6 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:25 -0400
Subject: [redhat] kabi: Adding symbol param_get_long (kernel/params.c)

Message-id: <bb4a4f670c63fc7043c1725d96288ab9ef4be2aa.1410510773.git.poros@redhat.com>
Patchwork-id: 96033
O-Subject: [PATCH RHEL7.1 145/233] kabi: Adding symbol param_get_long (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type long.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_get_long
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_get_long
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_get_long

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_get_long b/redhat/kabi/kabi-module/kabi_ppc64/param_get_long
new file mode 100644
index 0000000..d3596a0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_get_long
@@ -0,0 +1,2 @@
+#1-
+0xab1d6cc1 param_get_long vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_get_long b/redhat/kabi/kabi-module/kabi_s390x/param_get_long
new file mode 100644
index 0000000..d3596a0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_get_long
@@ -0,0 +1,2 @@
+#1-
+0xab1d6cc1 param_get_long vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_get_long b/redhat/kabi/kabi-module/kabi_x86_64/param_get_long
new file mode 100644
index 0000000..d3596a0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_get_long
@@ -0,0 +1,2 @@
+#1-
+0xab1d6cc1 param_get_long vmlinux EXPORT_SYMBOL
-- 
1.7.1